server/routerlicious/packages/services-telemetry/tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
There is a new pattern that is being used in FluidFramework around separate tsconfigs for test files. You can see an example of this in [R11s Lambdas](https://github.com/microsoft/FluidFramework/blob/main/server/routerlicious/packages/lambdas/src/test/tsconfig.json). It seems like it's mostly been applied to packages consumed by the client, so feel free to ignore, and we can do a larger refactor to align service packages with client patterns

PR for some context: #4626